agenta-web/src/components/AppSelector/modals/CreateAppStatusModal.tsx

```diff
@@ -67,18 +67,6 @@ const CreateAppStatusModal: React.FC<Props & React.ComponentProps<typeof Modal>>
         setIsDelayed(false)
     }
 
-    const onOk = (e: any) => {
```

Comment:
this shouldn't be removed, as the ok button is used to retry in error and timeout cases